steps/representative/representative-details/RepresentativeDetails.js

```diff
@@ -39,27 +48,23 @@ class RepresentativeDetails extends Question {
 
         return form({
 
-            firstName: text
-                .joi(
-                    fields.firstName.error.required,
-                    Joi.string().required()
-                ).joi(
-                    fields.firstName.error.invalid,
-                    Joi.string().trim().regex(firstName)
-                ),
-            lastName: text
-                .joi(
-                    fields.lastName.error.required,
-                    Joi.string().required()
-                ).joi(
-                    fields.lastName.error.invalid,
-                    Joi.string().trim().regex(lastName)
-                ),
-            organisation: text
-                .joi(
-                    fields.organisation.error.invalid,
-                    Joi.string().regex(whitelist).allow('')
-                ),
+            nameOrganisation: object({
+                firstName: text,
+                lastName: text,
+                organisation: text
```

Comment:
Can we change the name of this object to `name` and then the fields can become `first` and `last`. We can then reference these fields like: `name.first`, `name.last` and `name. organisation` instead of `nameOrganisation .organisation`.